snippets/language-html.cson

```diff
@@ -1,4 +1,4 @@
-'.text.html, .text.html .meta.scope.between-tag-pair, .text.html .punctuation.tag.begin':
+'.text.html, .text.html .meta.scope.between-tag-pair, .text.html .punctuation.tag.begin, .source.gfm':
```

Comment:
GFM has a whitelist of allowed HTML tags; the rest are stripped.
https://github.com/jch/html-pipeline/blob/6d223dc2f7e7f307d3bda8902e225c0d5ea0b2e8/lib/html/pipeline/sanitization_filter.rb#L44-L106

I am not aware if these snippets will override the gfm ones.